### PR TITLE
Fixes #1 Move to VS2017

### LINKED ISSUE #1: Move to VS2017
Move to VS2017

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

**Comment by msevestre** [comment]
Build fails on appveyor
https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-funcparser/build/3.0.0.2

**Comment by msevestre** [comment]
Probably needs sthg like that
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/develop/appveyor.yml#L2

**Comment by Yuri05** [comment]
yep, that was the problem

File: .gitignore [hunk 1]
```diff
@@ -250,3 +250,8 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+Build
```
Comment (msevestre) [inline .gitignore]: There is a section in the gitignore with excldue for build x32 and x64. Please enhance the section there
https://github.com/Yuri05/OSPSuite.FuncParser/blob/a34f73a4314a4ce5046b187bdf963ef363a07ac8/.gitignore#L13

File: .gitignore [hunk 2]
```diff
@@ -250,3 +250,8 @@ paket-files/
...
+/src/OSPSuite.FuncParser/ReleaseX64
+/tests/OSPSuite.FuncParser.Tests/ReleaseX64
+/00/*.7z
```
Comment (msevestre) [inline .gitignore]: I really don't like this kind of addition in gitignore . Why is this required?
